cli/src/pcluster/validators/common.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, raise_on_error=False):
     def _fail(self, message, level):
         raise ConfigValidationError
 
-    def _add_failure(self, message, level, fields=()):
+    def _add_failure(self, message: str, level: FailureLevel, fields=()):
```

Comment:
minor: why not also add type hint for fields? For example: `fields:List[Param]=()`

Reply:
Fixed, thanks